css_parser.gemspec

```diff
@@ -10,4 +10,10 @@ Gem::Specification.new name, CssParser::VERSION do |s|
   s.add_runtime_dependency 'addressable'
   s.files = Dir.glob("lib/**/*") + ["MIT-LICENSE"]
   s.license = "MIT"
+  
+  if s.respond_to?(:metadata)
+    s.metadata['changelog_uri'] = "https://github.com/premailer/css_parser/blob/master/CHANGELOG.md"
+    s.metadata['source_code_uri'] = "https://github.com/premailer/css_parser"
```

Comment:
it's already the homepage, isn't this kinda redundant ?

Reply:
It's really up to you. I like when it's explicit and it allows some nicer automation without guessing whether the homepage is also a source link. (there's also a `homepage_uri` metadata which I'm omitting)